src/screens/TermsConditionScreen.js

```diff
@@ -1,74 +1,93 @@
 import React from "react";
-import { StyleSheet, Button, Text, SafeAreaView } from "react-native";
+import { StyleSheet, Button, Text, SafeAreaView, View } from "react-native";
+
+import { TouchableOpacity } from "react-native-gesture-handler";
+
 
 import { AuthContext } from "../config/context";
 import { scale, moderateScale, verticalScale } from "../config/scaling";
 import colors from "../config/colors";
+import screenConfig from "../config/screen"
 
 const TermsConditionScreen = ({ navigation }) => {
   const { signIn } = React.useContext(AuthContext);
   return (
-    <SafeAreaView style={styles.container}>
-      <Text style={styles.titleText}>
-        {"\n"}
-        Terms and Condition
-      </Text>
-      <Text style={styles.text}>
-        Health Guard
-        {"\n"}
-      </Text>
+    <SafeAreaView style={screenConfig.container}>
+      <View style={styles.screenContainer}>
+
+        <Text style={screenConfig.subTitleText}>
+          Welcome to
+        </Text>
+        <Text style={screenConfig.titleText}>
+          Health Guard!
+        </Text>
+
+        <View style={styles.cardContainer}>
```

Comment:
![image](https://user-images.githubusercontent.com/45149474/117064091-b9d97a00-acf3-11eb-8d8f-b505374fb017.png)
It would be better to use ScrollView instead of View and put some height size on cardContainer